tests/jballerina-integration-test/src/test/resources/configurables/configurableProject/main/main.bal

```diff
@@ -38,10 +38,10 @@ public function main() {
     decimal result = 24.87;
     test:assertEquals(result, decimalVar);
 
-    test:assertEquals([1,2,3], intArr);
-    test:assertEquals([9.0, 5.6], floatArr);
-    test:assertEquals(["red", "yellow", "green" ], stringArr);
-    test:assertEquals([true, false, false, true], booleanArr);
+    test:assertEquals(<int[]>[1,2,3], intArr);
+    test:assertEquals(<float[]>[9.0, 5.6], floatArr);
+    test:assertEquals(<string[]>["red", "yellow", "green" ], stringArr);
+    test:assertEquals(<boolean[]>[true, false, false, true], booleanArr);
```

Comment:
Shouldn't these work without the casts?

Reply:
Without this we are getting `ambiguous type 'Comparable'`, `Comparable` is `anydata|error|Comparable[]|map<Comparable>`

Reply:
This also relates to https://github.com/ballerina-platform/ballerina-lang/pull/28062#discussion_r562162805